website/docs/d/qldb_ledger.html.markdown

```diff
@@ -0,0 +1,29 @@
+---
+layout: "aws"
+page_title: "AWS: aws_qldb_ledger"
+description: |-
+  Get information on a Amazon Quantum Ledger Database (QLDB)
+---
+
+# Data Source: aws_qldb_ledger
+
+Use this data source to get the ARN, deletion protection and permissions mode of a ledger in AWS Quantum Ledger Database 
```

Comment:
We are no longer including permissions mode in this data source, so we should omit that from the byline. Its generally easiest to say something along the lines of the following, which does not need updates when the data source has updates:

```md
Use this data source to fetch information about a Quantum Ledger Database.
```